src/main/java/org/pac4j/vertx/handler/impl/CallbackDeployingPac4jAuthHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.pac4j.vertx.handler.impl;
+
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import org.pac4j.core.config.Config;
+import org.pac4j.vertx.auth.Pac4jAuthProvider;
+
+/**
+ * @author jez
+ */
+public class CallbackDeployingPac4jAuthHandler extends RequiresAuthenticationHandler {
```

Comment:
Added issue #33 to improve the documentation both in-class and in the README for clarity around the use-case for this class (convenience, nothing more)